### ISSUE TITLE
pip install not working

### ISSUE THREAD

**Issue by pkill37** [bkerler/mtkclient#1132]
I cannot install mtk from the latest commit

```
$ pip3 install -r requirements.txt --break-system-packages
...
$ pip3 install . --break-system-packages
Defaulting to user installation because normal site-packages is not writeable
Processing /home/zzz/Desktop/mtkclient
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: pyusb in /usr/local/lib/python3.11/dist-packages/pyusb-1.2.1-py3.11.egg (from mtkclient==2.0.2) (1.2.1)
Requirement already satisfied: pycryptodome in /usr/local/lib/python3.11/dist-packages/pycryptodome-3.18.0-py3.11-linux-x86_64.egg (from mtkclient==2.0.2) (3.18.0)
Requirement already satisfied: pycryptodomex in /usr/local/lib/python3.11/dist-packages/pycryptodomex-3.18.0-py3.11-linux-x86_64.egg (from mtkclient==2.0.2) (3.18.0)
Requirement already satisfied: colorama in /usr/local/lib/python3.11/dist-packages/colorama-0.4.6-py3.11.egg (from mtkclient==2.0.2) (0.4.6)
Requirement already satisfied: shiboken6 in /usr/local/lib/python3.11/dist-packages/shiboken6-6.5.2-py3.11-linux-x86_64.egg (from mtkclient==2.0.2) (6.5.2)
Requirement already satisfied: pyside6 in /usr/local/lib/python3.11/dist-packages/PySide6-6.5.2-py3.11-linux-x86_64.egg (from mtkclient==2.0.2) (6.5.2)
Requirement already satisfied: mock in /usr/local/lib/python3.11/dist-packages/mock-5.1.0-py3.11.egg (from mtkclient==2.0.2) (5.1.0)
Requirement already satisfied: pyserial in /usr/local/lib/python3.11/dist-packages/pyserial-3.5-py3.11.egg (from mtkclient==2.0.2) (3.5)
Requirement already satisfied: fusepy in /home/zzz/.local/lib/python3.11/site-packages (from mtkclient==2.0.2) (3.0.1)
Requirement already satisfied: PySide6-Addons==6.5.2 in /usr/local/lib/python3.11/dist-packages/PySide6_Addons-6.5.2-py3.11-linux-x86_64.egg (from pyside6->mtkclient==2.0.2) (6.5.2)
Requirement already satisfied: PySide6-Essentials==6.5.2 in /usr/local/lib/python3.11/dist-packages/PySide6_Essentials-6.5.2-py3.11-linux-x86_64.egg (from pyside6->mtkclient==2.0.2) (6.5.2)
Building wheels for collected packages: mtkclient
  Building wheel for mtkclient (pyproject.toml) ... done
  Created wheel for mtkclient: filename=mtkclient-2.0.2-py3-none-any.whl size=253853 sha256=4edb550fdc00102ddf638bd01e62cb6f4dcf9fd66a6abe0063c2945f1b5d0c2e
  Stored in directory: /tmp/pip-ephem-wheel-cache-2pjgisvt/wheels/ae/88/a5/677fa177b894380f0942725fd5cb06238a757939a7e537b304
Successfully built mtkclient
Installing collected packages: mtkclient
Successfully installed mtkclient-2.0.2
```
```
$ mtk
Traceback (most recent call last):
  File "/usr/local/bin/mtk", line 5, in <module>
    from mtk import main
ModuleNotFoundError: No module named 'mtk'
```

How is it supposed to be installed? I'm on Debian 12

**Comment by pkill37** [comment]
Same from a clean virtual env

**Comment by pkill37** [comment]
Maybe you would rather deprecate installation and recommend people to execute `python mtk.py ...`?

**Comment by bkerler** [comment]
Solved with commit [168aa1d](https://github.com/bkerler/mtkclient/commit/168aa1d464ccbfdf5bc6d7105ef53e1bcd68cc36)
